simple-examples/Examples.hs

```diff
@@ -0,0 +1,4 @@
+import System.Exit
+
+main :: IO ()
+main = exitSuccess
```

Comment:
This is a curious idiom. Maybe document.

Reply:
A cleaner choice would be to use Cabal internal libraries. See https://www.haskell.org/cabal/users-guide/developing-packages.html#sublibs.